.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -15,28 +15,46 @@ This link describes the WinUI feature/API proposal process:
 https://github.com/Microsoft/microsoft-ui-xaml/blob/master/docs/feature_proposal_process.md
 -->
 
-<!-- Add a title for your feature or API proposal. Please be short and descriptive -->
 # Proposal: [your title here] 
+<!-- Add a title for your feature or API proposal. Please be short and descriptive -->
 
 ## Summary
-<!-- Please include a couple sentences summarizing your feature or API proposal -->
+<!-- Include 1-2 sentences summarizing your feature or API proposal -->
 
 ## Rationale
-<!-- Please describe WHY the feature should be added to WinUI for all developers and users.
-If applicable you can also describe how it aligns to the current WinUI roadmap and priorities:
+<!-- Fill in the table below to describe WHY the feature should be added to WinUI for all developers and users.
+Proposals often have multiple motives for why we should do the work, so list each one as a separate row item.
+If applicable you can also describe how it aligns to the current WinUI roadmap and priorities in a separate paragraph:
 https://github.com/Microsoft/microsoft-ui-xaml/blob/master/docs/roadmap.md
 -->
+| Motive |
+| :----- |
+| {list each motivating reason as a separate row} |
```

Comment:
```suggestion
| <!-- list each motivating reason as a separate row --> |
```

Reply:
Unless you just intend for the text to be deleted. The brackets threw me.

Reply:
Yeah, I put the brackets there to indicate that people should replace that text with their own actual real motivators.  Unsure if there's a better way to stylistically represent that, rather than curly braces.  (I had thought of using Italics, but then I'm worried that people will put all the rows of the table in Italics :))